Globex-Database/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM mcr.microsoft.com/mssql/server
+FROM postgres:15.1
+
+RUN localedef -i es_ES -c -f UTF-8 -A '/usr/share/locale/locale.alias' es_ES

```

Comment:
Why are we changing the locale to Spanish here? (seems to run file without this line)
```suggestion

```